tests/test_generic_plotter.py

```diff
@@ -20,12 +20,17 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 """Test module for the generic plotter."""
+import os
 from pathlib import Path
 
 import numpy as np
+import pytest
 import pyvista as pv
 
 from ansys.tools.visualization_interface import ClipPlane, MeshObjectPlot, Plotter
+from ansys.tools.visualization_interface.backends.pyvista import PyVistaBackend
+
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
```

Comment:
Align env var naming... Looks like in your workflow this is called ``IN_GITHUB_ACTIONS`` whereas here you are trying to retrieve ``GITHUB_ACTIONS``. Does that mean that it IS being executed on CI/CD?

Reply:
Hmm.. let me test this

Reply:
Aligned, for some reason it was being skipped anyway. In my local tests the test is working and not being skipped.